packages/rs-dpp/src/identity/validation/public_keys_validator.rs

```diff
@@ -173,6 +173,8 @@ impl<T: BlsModule> PublicKeysValidator<T> {
     ) -> Result<ValidationResult<()>, NonConsensusError> {
         self.public_key_schema_validator.validate(public_key)
     }
+
+    pub fn validate_state(&self) {}
```

Comment:
probably TODO or it should be removed.

Reply:
Oh this one actually shouldn't be there at all